functional_tests/grpc/02-mp-log-image-sequence.yea

```diff
@@ -0,0 +1,42 @@
+id: 0.grpc.02-mp-log-image-sequence
+plugin:
+    - wandb
+depend:
+    requirements:
+        - numpy
+        - pillow
+tag:
+    shard: grpc
+parametrize:
+    permute:
+        - :yea:start_method:
+            - spawn
+            - forkserver
+var:
+    - history_len:
+        :fn:len: :wandb:runs[0][history]
+    - filenames_0_len:
+         :fn:len: :wandb:runs[0][history][0][media][filenames]
+    - filenames_1_len:
+         :fn:len: :wandb:runs[0][history][1][media][filenames]
+assert:
+    - :wandb:runs_len: 1
+
+    - :wandb:runs[0][history][0][_step]: 0
+    - :wandb:runs[0][history][1][_step]: 1
+
+    - :history_len: 2
+
+    - :wandb:runs[0][history][0][media][_type]: images/separated
+    - :wandb:runs[0][history][0][media][width]: 2
+    - :wandb:runs[0][history][0][media][height]: 2
+    - :wandb:runs[0][history][0][media][count]: 3
+    - :filenames_0_len: 3
```

Comment:
IMO, since we're relying on the sha to differentiate the files, and including them in the file names, it would be nice to have the test check that each filename is correct (has the expected sha based on the test).

Reply:
I tried to do it, but it hard to figure out the hash ahead of time (without actually reimplementing the hashing logic)...
do you know if there is an easy way to get the sha information?